### ISSUE TITLE
HARPS optics upgrade

### ISSUE THREAD

**Issue by megbedell** [megbedell/wobble#14]
HARPS had a major upgrade in July 2015 where fibers were replaced (see [this Messenger article](https://www.eso.org/sci/publications/messenger/archive/no.162-dec15/messenger-no162-9-15.pdf) for details). In the derived RVs, both from the HARPS pipeline and from wobble, this manifests as an offset of about -15 m/s after the upgrade.

Long-term I think this means we should treat the pre- and post-upgrade spectra as coming from different instruments with different stellar (and telluric??) models. This does mean the post-upgrade data won't be very useful because we don't have many epochs to use. A short-term hack might be to just build an effective post-upgrade RV offset as a free parameter in the model we use to get the final time-series RVs. Or we could just side-step the problem for now and only use pre-upgrade spectra in Paper 1.

**Comment by davidwhogg** [comment]
I agree we should treat these as different instruments (for now). Eventually, when we have a method for permitting tellurics and spectra to change with time, we might be able to do everything at once, but that time is in the future.
